tests/sources/fixtures/redis/fixture.py

```diff
@@ -0,0 +1,47 @@
+#
+# Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+# or more contributor license agreements. Licensed under the Elastic License 2.0;
+# you may not use this file except in compliance with the Elastic License 2.0.
+#
+# ruff: noqa: T201
+import os
+import random
+
+import redis.asyncio as redis
+
+from tests.commons import WeightedFakeProvider
+
+DATA_SIZE = os.environ.get("DATA_SIZE", "small").lower()
+_NUM_TABLES = {"small": 2, "medium": 4, "large": 6}
```

Comment:
How much does this affect time of the run? Ideally, large should take 20+ minutes, while small should take <1 minute

Reply:
Yes, it is taking less than a minute to complete this.